api/src/models/AudioFile.test.ts

```diff
@@ -0,0 +1,21 @@
+import AudioFile from "./AudioFile";
+import Config from "../models/Config";
+
+jest.mock("../models/Config");
+
+describe("Audio", () => {
+    let audio: AudioFile;
+    let config;
+    beforeEach(() => {
+        config = {
+            ffmpegPath: "/usr/bin/ffmpeg"
```

Comment:
It might be better to use a less realistic value here, like `/foo/ffmpeg`, so that if the test somehow triggers real behavior, it doesn't accidentally run a process that we don't really want to run. Or it might be better not to set this at all in the `beforeEach` but wait until we're testing a method that requires the setting, and add it as part of the test setup. (Whether to do this sort of thing globally or locally is kind of a judgment call -- depends on how many tests rely on the same input, etc.).

Reply:
fixed